Libstapsdt.cs

```diff
@@ -1,76 +1,98 @@
-﻿namespace Libstapsdt;
+﻿using System.Runtime.CompilerServices;
```

Comment:
Throw a `dotnet format` at it?

Reply:
I think better to go with code styles in `.editorconfig` and then have them enforced, which should take care of such inconsistencies. I find `dotnet format` may give you consistency but hinders readability.

For now, I've addressed this with 29d261fd787a2556eec3aa8984ff590ebf608287.

Reply:
Gotcha. Looks like `dotnet format` [also understands](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-format#description) `.editorconfig` - is it the tool you normally use for code autoformatting?

Reply:
I personally avoid auto-formatters because I subscribe to the school that readability is the responsibility of the writer and requires thought and care, especially with some modern syntax.